src/summaries/level-two-routes.component.tsx

```diff
@@ -28,6 +29,21 @@ export const levelTwoRoutes: PatientChartRoute[] = [
     component: AllergyCardLevelThree,
     name: "Allergy"
   },
+  {
+    url: "/patient/:patientUuid/chart/conditions",
+    component: ConditionsBriefSummary,
```

Comment:
this component was renamed to ConditionsOverview in master.

Reply:
ConditionsBriefSummary is the level-two component. ConditionsOverview is the master card.